crates/core/src/client_events/mod.rs

```diff
@@ -439,6 +458,28 @@ async fn process_open_request(
                             // Since there are no remote peers, this will complete locally
                             if let Err(err) = put::request_put(&op_manager, op).await {
                                 tracing::error!("Local PUT request error: {}", err);
+
+                                // Notify client of error via result router (actor mode)
+                                if let Some(router_tx) = &op_manager.result_router_tx {
+                                    let error_response = Err(ErrorKind::OperationError {
+                                        cause: format!("PUT operation failed: {}", err).into(),
+                                    }
+                                    .into());
+
+                                    let router_tx_clone = router_tx.clone();
+                                    let op_id_clone = op_id;
+                                    tokio::spawn(async move {
```

Comment:
why are we spawning just to send this? can't we just send inside the same task seems overkill and probably not more efficient overall?